pkg/kubelet/dockertools/manager.go

```diff
@@ -384,11 +384,14 @@ func (dm *DockerManager) inspectContainer(id string, podName, podNamespace strin
 
 		terminationMessagePath := containerInfo.TerminationMessagePath
 		if terminationMessagePath != "" {
-			if path, found := iResult.Volumes[terminationMessagePath]; found {
-				if data, err := ioutil.ReadFile(path); err != nil {
-					message = fmt.Sprintf("Error on reading termination-log %s: %v", path, err)
-				} else {
-					message = string(data)
+			for _, mount := range iResult.Mounts {
```

Comment:
This fixed the termination message path. (See #23658)
Only with it, the termination message path can work correctly, but unfortunately, it hasn't been merged.
To test termination message, we should let it work right first, so I cherry-pick that commit here. Will rebase after #23658 get merged.